src/engraving/rendering/score/slurtielayout.cpp

```diff
@@ -2880,6 +2880,7 @@ bool SlurTieLayout::isDirectionMixture(const Chord* c1, const Chord* c2, LayoutC
 
 bool SlurTieLayout::shouldHideSlurSegment(SlurSegment* item, LayoutContext& ctx)
 {
+    UNUSED(ctx);
```

Comment:
Whenever we can, let's just remove unused parameters, instead of using this macro